src/api/config/routes.rb

```diff
@@ -610,8 +612,12 @@ def self.matches?(request)
     put '/mail_handler' => 'mail_handler#upload'
 
     ### /public
-
     controller :public do
+      # first the routes where the mime type does not matter
+      ### /public
+      get 'public/build/:project(/:repository(/:arch(/:package(/:file))))' => 'public#build', constraints: cons, as: :public_build
```

Comment:
Ok, will move the build route to the bottom as there is no other method which can infer...